cpp-compile.Dockerfile

```diff
@@ -71,68 +74,67 @@ ENV GCC_CONFIG="\
     --enable-languages=c,c++,fortran \
     --disable-multilib \
 "
-RUN set -x                                                                     && \
-    tar xf $GCC_TGZ                                                            && \
-    cd $GCC                                                                    && \
-    contrib/download_prerequisites                                             && \
-    ./configure -v $GCC_CONFIG                                                 && \
-    make -j$(($(nproc)+1))                                                     && \
-    make install-strip                                                         && \
-    cd ..                                                                      && \
-    rm $GCC $GCC_TGZ $GCC_TGZ.sig -rf                                          && \
-    echo '/usr/local/lib64' > /etc/ld.so.conf.d/local-lib64.conf               && \
-    ldconfig -v                                                                && \
-    dpkg-divert --divert /usr/bin/gcc.orig --rename /usr/bin/gcc               && \
-    dpkg-divert --divert /usr/bin/g++.orig --rename /usr/bin/g++               && \
-    dpkg-divert --divert /usr/bin/gfortran.orig --rename /usr/bin/gfortran     && \
-    update-alternatives --install /usr/bin/cc cc /usr/local/bin/gcc 999        && \
+RUN set -x && \
+    tar xf $GCC_TGZ && \
+    cd $GCC && \
+    contrib/download_prerequisites && \
+    ./configure -v $GCC_CONFIG && \
+    make -j$(($(nproc)+1)) && \
+    make install-strip && \
+    cd .. && \
+    rm $GCC $GCC_TGZ $GCC_TGZ.sig -rf && \
+    echo '/usr/local/lib64' > /etc/ld.so.conf.d/local-lib64.conf && \
+    ldconfig -v && \
+    dpkg-divert --divert /usr/bin/gcc.orig --rename /usr/bin/gcc && \
+    dpkg-divert --divert /usr/bin/g++.orig --rename /usr/bin/g++ && \
+    dpkg-divert --divert /usr/bin/gfortran.orig --rename /usr/bin/gfortran && \
+    update-alternatives --install /usr/bin/cc cc /usr/local/bin/gcc 999 && \
     update-alternatives --install /usr/bin/c++ c++ /usr/local/bin/g++ 999
 
-## Build boost 1.70.0
-RUN set -x                                                                     && \
-    tar xf $BOOST_TGZ                                                          && \
-    cd $BOOST                                                                  && \
-    ./bootstrap.sh --with-libraries=graph,headers,program_options,regex        \
-    --prefix=/usr/local                                                        && \
-    ./b2 install                                                               && \
-    cd ..                                                                      && \
+## Build boost
+RUN set -x && \
+    tar xf $BOOST_TGZ && \
+    cd $BOOST && \
+    ./bootstrap.sh --with-libraries=graph,headers,program_options,regex --prefix=/usr/local && \
```

Comment:
I can we install all boost libraries please. We need them for the webserver. I recommend `--without-libraries=python` instead of the `--with-libraries...`